f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml

```diff
@@ -750,13 +752,11 @@ Dow.prototype = {
   getXaasServices: function(packageId, mappedPeriods) {
     try {
       var table, configs;
+      // Q: if compute/database/storage do not have selcted services
+      // How are they being stored? 
+      // And how are different configs being stored for each?
+      // this will not capture those services
```

Comment:
Compute/Database/Storage instances each have their own tables that extend `Environment Instance`.  Get at them via the Acquisition Package reference in the base table.

![image](https://user-images.githubusercontent.com/77642966/210686458-8f6827bf-a3db-4856-88ca-b7702703509e.png)

Reply:
Yeah, that's how I was getting at them, but I structured the data incorrectly because I thought both the instances and the selected services were being used to store information. So that needs to be refactored.

Reply:
Updated  `xaasOfferings` to return 

```typescript
xaasOffering: {
  computeInstances: [];
  databaseInstances: [];
  storageInstances: [];
  generalInstances: [];
  selectedServiceInstances: [];
}
```